process/prefixer.go

```diff
@@ -1,83 +1,73 @@
 package process
 
 import (
-	"bytes"
 	"io"
 	"regexp"
-	"unicode/utf8"
 )
 
-// This regex matches "Erase in Line" escape characters:
+// This regex matches either newline, or one of the "Erase in Line" escape
+// sequences:
 // [K Clear from cursor to the end of the line
 // [1K Clear from cursor to beginning of the line
 // [2K Clear entire line
-var lineBreakEscapeCharRegex = regexp.MustCompile(`^\[[012]?K`)
+var lineBreakEscapeRE = regexp.MustCompile(`\n|\x1b\[[012]?K`)
 
+// Prefixer inserts prefixes generated by a callback before each line.
```

Comment:
Maybe make a mention here that you should always Flush()

Reply:
Done